### PR TITLE
feat: sanitize descriptions on help page

### LINKED ISSUE #102: Sanitize descriotions on HelpPage
It should be possible to have more flexible desciptions at the help page containing links, etc.
Therefore the values in `messages.json` should be sanitized, so that it's allowed to configure HTML inside those messages.

### PR DESCRIPTION
**Pull request by d-koppenhagen**
allow to customize description content by sanitizing it's content

closes #102

### CODE REVIEW

File: src/components/PageHelp/PageHelp.tsx [hunk 1]
```diff
@@ -21,19 +29,29 @@ const PageHelp: React.FC<Props> = ({ leaving, onLeave }) => {
...
-              <h3>{headline}</h3>
-              {values.map((element, index) => (
-                <p key={index}>{element}</p>
-              ))}
-            </React.Fragment>
+              <React.Fragment key={headline}>
+                <h3>{headline}</h3>
+                {values.map((element, index) => {
+                  const content = sanitizeHtml(element, {
+                    allowedTags: [ 'b', 'i', 'em', 'strong', 'a' ],
```
Comment (tclift) [inline src/components/PageHelp/PageHelp.tsx:36]: Any harm in adding `ul`, `ol` and `li` at the same time? We use lists on our (forked) About page.

File: src/components/PageHelp/PageHelp.tsx [hunk 2]
```diff
@@ -21,19 +29,29 @@ const PageHelp: React.FC<Props> = ({ leaving, onLeave }) => {
...
+                    allowedAttributes: {
+                      'a': [ 'href' ]
```
Comment (tclift) [inline src/components/PageHelp/PageHelp.tsx:38]: I haven't checked if links open in new tabs by default after this change is applied... but if not, any harm in adding `target` at the same time (for e.g., `<a href="..." target="_blank">...</a>`)?

**Comment by tclift** [comment]
Thanks @d-koppenhagen. Your three open pull requests should make it possible for us to migrate from our forked radar to using it as a dependency, and bring back some of the configuration that was lost in the v2 upgrade. Much appreciated.

File: src/components/PageHelp/PageHelp.tsx (hunk 1 above)
Comment (d-koppenhagen) [reply src/components/PageHelp/PageHelp.tsx]: Thanks for the suggestion, I added the elements

File: src/components/PageHelp/PageHelp.tsx (hunk 2 above)
Comment (d-koppenhagen) [reply src/components/PageHelp/PageHelp.tsx]: Thanks for the suggestion, I added this attribute

File: src/components/PageHelp/PageHelp.tsx [hunk 3]
```diff
@@ -21,19 +29,29 @@ const PageHelp: React.FC<Props> = ({ leaving, onLeave }) => {
...
+              <React.Fragment key={headline}>
+                <h3>{headline}</h3>
+                {values.map((element, index) => {
+                  const content = sanitizeHtml(element, {
+                    allowedTags: ['b', 'i', 'em', 'strong', 'a', 'ul', 'ol', 'li'],
+                    allowedAttributes: {
+                      'a': ['href', 'target']
+                    },
+                  });
+                  console.log(content);
```
Comment (bastianccm) [inline src/components/PageHelp/PageHelp.tsx:41]: Do we need the `console.log` here?

**Comment by bastianccm** [comment]
...while I'm not a big fan of `sanitizeHtml` and stuff I think we can assume that the description json can be trusted. So, LGTM, I'll merge after feedback on `console.log`

File: src/components/PageHelp/PageHelp.tsx (hunk 3 above)
Comment (d-koppenhagen) [reply src/components/PageHelp/PageHelp.tsx:41]: Sure! good catch

Comment (d-koppenhagen) [reply src/components/PageHelp/PageHelp.tsx:41]:
```suggestion
```

**Comment by d-koppenhagen** [comment]
@bastianccm me neither. But I was thinking the same: it should be totally fine to trust the descriptions json. Another approach would be to refactor the whole help page to use a markdown file als template.
Probably this would be a more flexible way for providing a description.
But maybe we can discuss this in another issue in advance.

**Comment by bastianccm** [comment]
It's fine :) appreciate the work!

**Pull request merged**